packages/widgets/src/Pagination/PageSummary.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import propTypes from '@dhis2/prop-types'
+import i18n from '@dhis2/d2-i18n'
+import { spacers } from '@dhis2/ui-constants'
+
+import { sectionStyles } from './Pagination.styles.js'
+
+const getItemRange = (page, pageSize, total) => {
+    const firstItem = total === 0 ? 0 : (page - 1) * pageSize + 1
+    const lastItem = Math.min(firstItem + pageSize - 1, total)
```

Comment:
Could you write comments about what's happening here, I have a hard time understanding what's going on without reading into the code and it's logic

Reply:
Yeah that seems nice to have 👍

Reply:
aba32bf